utilities/titandb/titan_db_test.cc

```diff
@@ -252,6 +252,26 @@ TEST_F(TitanDBTest, Snapshot) {
     db_->ReleaseSnapshot(snapshot);
 }
 
+TEST_F(TitanDBTest, ReadAfterDropCF) {
```

Comment:
You can mark it as disabled for now before fixed: `DISABLED_ReadAfterDropCF`. This will avoid gtest from running the test by default.